ui/src/app/views/invitation-key/component.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import {mount} from 'enzyme';
+import * as React from 'react';
+
+import InvitationKeyReact, {InvitationKeyProps} from './component';
+
+import {AlertDanger} from 'app/components/alert';
+import {Button} from 'app/components/buttons';
+import {FormInput} from 'app/components/inputs';
+import {profileApi, registerApiClient} from 'app/services/swagger-fetch-clients';
+import {ProfileApi} from 'generated/fetch';
+import {ProfileApiStub} from 'testing/stubs/profile-api-stub';
+
+describe('InvitationKeyComponent', () => {
+  let props: InvitationKeyProps;
+  const onInvitationVerify = jest.fn();
+
+
+  const component = () => {
+    return mount<InvitationKeyReact>
```

Comment:
Please remove the newline, possibly this is something that can be automated by our linter, but we really should avoid putting a newline between method name and paren to invoke that method.

Also is the generic type actually necessary here?